packages/frontend/webapp/src/components/CarteSite/CarteSite.vue

```diff
@@ -86,7 +86,7 @@
         </main>
 
         <footer class="mt-auto">
-            <Link :to="`/site/${shantytown.id}`" @click="routeToDetailsPage"
+            <Link @click="routeToDetailsPage"
```

Comment:
il y avait ici une redondance entre le :to="..." et le @click="..." que je supprime au passage